.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/*
 *.iml
 target/*
+Main.java
+MyTests.java
+TestClass.java
```

Comment:
let's remove this
in the checklist we meant that your pushed code should not contain Main class or main() method
in other words, you should make changes in class Rectangle only